mldsa/ntt.c

```diff
@@ -212,8 +222,10 @@ void invntt_tomont(int32_t a[MLDSA_N])
   for (j = 0; j < MLDSA_N; ++j)
```

Comment:
Please adjust the comment above the loop to accurately reflect the bounds - mention that it performs Montgomery multiplication by `mont^2/256`